src/Header/Header.scss

```diff
@@ -6,15 +6,14 @@ header {
   padding: 1rem 0;
   position: sticky;
   top: 0;
-  z-index: 1;
+  z-index: 5;
```

Comment:
Just out of curiosity, why did you make this change?

Reply:
@mec07  Because there was an issue with the contact form being on top of the header on desktop. 
Thanks for pointing it out  😄